home/lcd.asm

```diff
@@ -35,13 +35,6 @@ LCDMusicPlayer::
 	ld [oamSprite01YCoord], a
 	ld [oamSprite02YCoord], a
 
-	ldh a, [hMPState]
```

Comment:
(This was a leftover from https://github.com/Rangi42/polishedcrystal/commit/7206aecb7021adbe412ecb3719136d7da76b2c45.)